site/about/glossary/_validmind.qmd

```diff
@@ -3,7 +3,7 @@
 These two features are intertwined and work in tandem to help streamline your model lifecycle.
 
 <span id="framework">{{< var validmind.developer >}} ({{< var vm.developer >}})</span>
-: An open-source^[**GitHub:** [validmind / developer-framework](https://github.com/validmind/developer-framework/)] suite of documentation tools and test suites designed to document models, test models for weaknesses, and identify overfit areas. Enables automating the generation of model documentation by uploading documentation and test results to the {{< var validmind.product >}}.
+: An open-source^[**{{< var vm.product >}} GitHub:** [developer-framework](https://github.com/validmind/developer-framework/)] suite of documentation tools and test suites designed to document models, test models for weaknesses, and identify overfit areas. Enables automating the generation of model documentation by uploading documentation and test results to the {{< var validmind.platform >}}.
```

Comment:
Might be good to add something like `In versions before x, the ({{< var vm.developer >}} was known as the ValidMind Developer Framework` — kind of like a mental redirect for customers who have heard us talk about the old name off and on.

Reply:
I'm going to wrap this into sc-6648 (the release notes) as we need to wait until the repo is renamed.